Makefile

```diff
@@ -35,6 +35,9 @@ $(BUILDDIR)/apple/Tun2socks.xcframework: $(BUILDDIR)/ios/Tun2socks.xcframework $
 	find $^ -name "Tun2socks.framework" -type d | xargs -I {} echo " -framework {} " | \
 		xargs xcrun xcodebuild -create-xcframework -output "$@"
 
+browser:
```

Comment:
Let's skip in https://github.com/Jigsaw-Code/outline-client/blob/master/src/tun2socks/build.action.mjs instead, so it doesn't even reach the make step and the per-platform dispatching is centralized in one place.